[18.0] [MIG] l10n_it_vat_settlement_communication

odooNextev opened the pull request.

eLBati committed:
Renamed to l10n_it_vat_statement_communication
Make terms translatable
Make translations
IMP README

Merge l10n_it_vat_statement_communication_bridge
ADD readme dir and translations
IMP Quadro VP view with better selection of statements
REF compute_from_liquidazioni for extensibility
ADD chatter

eLBati committed:
porting l10n_it_vat_statement_communication to 12

Commit:
FIX l10n_it_vat_statement_communication CodiceFornitura is fixed IVP18

Commit:
[12.0]IMP l10n_it_vat_statement_communication aggiunti parametri per
escludere operatione e tax

eLBati committed:
IMP README for vsc_exclude_operation and vsc_exclude_vat (#6)

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 98.5% (130 of 132 strings)

Translation: l10n-italy-12.0/l10n-italy-12.0-l10n_it_vat_statement_communication
Translate-URL: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_vat_statement_communication/it/

eLBati committed:
FIX l10n_it_vat_statement_communication: set previous year credits in the right field

primes2h committed:
Translated using Weblate (Italian)

Currently translated at 97.0% (131 of 135 strings)

Translation: l10n-italy-12.0/l10n-italy-12.0-l10n_it_vat_statement_communication
Translate-URL: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_vat_statement_communication/it/

eLBati committed:
IMP l10n_it_vat_statement_communication adding down payment handling

eLBati committed:
VP14 non deve essere compilato dai contribuenti trimestrali di cui all’art. 7 del d.P.R. 14 ottobre 1999, n.542, relativamente al 4° trimestre

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (135 of 135 strings)

Translation: l10n-italy-12.0/l10n-italy-12.0-l10n_it_vat_statement_communication
Translate-URL: https://translation.odoo-community.org/projects/l10n-italy-12-0/l10n-italy-12-0-l10n_it_vat_statement_communication/it/

eLBati committed:
FIX l10n_it_vat_statement_communication adding "Down payment computation method" to be set when "Down payment due" is set

eLBati committed:
IMP l10n_it_vat_statement_communication setting correct XML file name

eLBati committed:
FIX l10n_it_vat_statement_communication: TypeError: Argument must be bytes or unicode, got 'int'

l10n_it_vat_statement_communication/models/comunicazione_liquidazione.py", line 268, in _export_xml_get_frontespizio
    x1_2_1_5_UltimoMese.text = self.last_month
  File "src/lxml/lxml.etree.pyx", line 1031, in lxml.etree._Element.text.__set__ (src/lxml/lxml.etree.c:53218)
  File "src/lxml/apihelpers.pxi", line 715, in lxml.etree._setNodeText (src/lxml/lxml.etree.c:24413)
  File "src/lxml/apihelpers.pxi", line 703, in lxml.etree._createTextNode (src/lxml/lxml.etree.c:24276)
  File "src/lxml/apihelpers.pxi", line 1441, in lxml.etree._utf8 (src/lxml/lxml.etree.c:31470)
TypeError: Argument must be bytes or unicode, got 'int'

eLBati committed:
FIX l10n_it_vat_statement_communication: I contribuenti che hanno optato per la liquidazione trimestrale ai sensi dell’art. 7 del D.P.R. n. 542/99 devono indicare “5” per il quarto trimestre

eLBati committed:
FIX l10n_it_vat_statement_communication: interessi_dovuti amount must come from interests_debit_vat_amount

Commit:
merge PR 1264 1717 1728 1737 1739

eLBati committed:
FIX l10n_it_vat_statement_communication when setting declarant_fiscalcode_company
File "/home/odoo/custom-addons/OCA/l10n-italy/l10n_it_vat_statement_communication/models/comunicazione_liquidazione.py", line 291, in _export_xml_get_frontespizio self.declarant_fiscalcode_company.code AttributeError: 'str' object has no attribute 'code'

eLBati committed:
FIX license LGPL-3 conflicts

eLBati committed:
IMP l10n_it_vat_statement_communication: interessi quarto trimestre in liquidazioni trimestrali

eLBati committed:
l10n_it_vat_statement_communication: retrieve computation method from VAT statement

sergiocorato committed:
[FIX][12.0] l10n_it_vat_statement_communication file export

eLBati committed:
[IMP] l10n_it_vat_statement_communication: black, isort, prettier

eLBati committed:
porting l10n_it_vat_statement_communication to 14.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 95.7% (135 of 141 strings)

Translation: l10n-italy-14.0/l10n-italy-14.0-l10n_it_vat_statement_communication
Translate-URL: https://translation.odoo-community.org/projects/l10n-italy-14-0/l10n-italy-14-0-l10n_it_vat_statement_communication/it/

TonyMasciI committed:
[MIG] l10n_it_vat_statement_communication: Migration to 16.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (138 of 138 strings)

Translation: l10n-italy-16.0/l10n-italy-16.0-l10n_it_vat_statement_communication
Translate-URL: https://translation.odoo-community.org/projects/l10n-italy-16-0/l10n-italy-16-0-l10n_it_vat_statement_communication/it/

SirAionTech committed:
Updated from template

SirAionTech committed:
[REF] l10n_it_vat_statement_communication: Update repo template

SirAionTech committed:
[REF] l10n_it_vat_statement_communication: Reformatted after template update

SirAionTech committed:
Updated from template

SirAionTech committed:
[COV] l10n_it_vat_statement_communication: Execute test once

This also triggers the following warning while executing tests:
> WARNING <db_name> odoo.tests.common: A tests should be either at_install or post_install, which is not the case of <class 'odoo.addons.l10n_it_vat_statement_communication.tests.test_vat_statement_communication.VatStatementCommunicationCase'>

matteoopenf committed:
[FIX]l10n_it_vat_statement_communication: fix file export

Avoid to output value at zero according to AdE specification

In exported xml file fix the character encoding value in doctype
declaration

odooNextev committed:
[IMP] l10n_it_vat_statement_communication: pre-commit auto fixes

primes2h commented:
/ocabot migration l10n_it_vat_statement_communication

odooNextev committed:
[MIG] l10n_it_vat_statement_communication: Migration to 18.0

primes2h reviewed (changes requested):
https://github.com/OCA/l10n-italy/issues/4510

odooNextev commented:
> #4510

@primes2h quindi rinomino il modulo in `l10n_it_vat_settlement_communication` e cerco tutti i riferimenti nel codice?

primes2h commented:
> > #4510
> 
> @primes2h quindi rinomino il modulo in `l10n_it_vat_settlement_communication` e cerco tutti i riferimenti nel codice?

:+1:

eLBati committed:
Adapt to l10n_it_account_vat_period_end_settlement
Adapt to RC tax structure, using vsc_exclude_active_operation and vsc_exclude_passive_operation
REF tests

VBNext reviewed (approved):
Funzionale OK

primes2h commented:
> > > #4510
> > 
> > 
> > @primes2h quindi rinomino il modulo in `l10n_it_vat_settlement_communication` e cerco tutti i riferimenti nel codice?
> 
> 👍

@odooNextev

eLBati committed:
Rename l10n_it_vat_statement_communication > l10n_it_vat_settlement_communication

eLBati commented:
@primes2h grazie , fatto

eLBati commented:
Testata la rinomina

/ocabot merge nobump
